utils/market.ts

```diff
@@ -0,0 +1,48 @@
+import { BN } from "@polkadot/util";
+import { api, Extrinsic } from "./setup";
+
+export class Market {
+  static createPool(
+    firstAsset: BN,
+    firstAssetAmount: BN,
+    secondAsset: BN,
+    secondAssetAmount: BN,
+    kind: any = "Xyk",
```

Comment:
this will be an enum, but its fine for now :)

Reply:
I see, I'll try to remmeber it